
<!doctype html>
<html lang="en" class="no-js">
  <head>
    
      <meta charset="utf-8">
      <meta name="viewport" content="width=device-width,initial-scale=1">
      
      
      
      
      
        <link rel="next" href="contributing.html">
      
      
      <link rel="icon" href="assets/images/favicon.png">
      <meta name="generator" content="mkdocs-1.5.3, mkdocs-material-9.5.17">
    
    
      
        <title>Awesome BIDS</title>
      
    
    
      <link rel="stylesheet" href="assets/stylesheets/main.bcfcd587.min.css">
      
        
        <link rel="stylesheet" href="assets/stylesheets/palette.06af60db.min.css">
      
      


    
    
      
    
    
      
        
        
        <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
        <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Roboto:300,300i,400,400i,700,700i%7CRoboto+Mono:400,400i,700,700i&display=fallback">
        <style>:root{--md-text-font:"Roboto";--md-code-font:"Roboto Mono"}</style>
      
    
    
    <script>__md_scope=new URL(".",location),__md_hash=e=>[...e].reduce((e,_)=>(e<<5)-e+_.charCodeAt(0),0),__md_get=(e,_=localStorage,t=__md_scope)=>JSON.parse(_.getItem(t.pathname+"."+e)),__md_set=(e,_,t=localStorage,a=__md_scope)=>{try{t.setItem(a.pathname+"."+e,JSON.stringify(_))}catch(e){}}</script>
    
      

    
    
    
  </head>
  
  
    
    
      
    
    
    
    
    <body dir="ltr" data-md-color-scheme="default" data-md-color-primary="indigo" data-md-color-accent="indigo">
  
    
    <input class="md-toggle" data-md-toggle="drawer" type="checkbox" id="__drawer" autocomplete="off">
    <input class="md-toggle" data-md-toggle="search" type="checkbox" id="__search" autocomplete="off">
    <label class="md-overlay" for="__drawer"></label>
    <div data-md-component="skip">
      
        
        <a href="#awesome-bids" class="md-skip">
          Skip to content
        </a>
      
    </div>
    <div data-md-component="announce">
      
    </div>
    
    
      

  

<header class="md-header md-header--shadow" data-md-component="header">
  <nav class="md-header__inner md-grid" aria-label="Header">
    <a href="index.html" title="Awesome BIDS" class="md-header__button md-logo" aria-label="Awesome BIDS" data-md-component="logo">
      
  
  <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M12 8a3 3 0 0 0 3-3 3 3 0 0 0-3-3 3 3 0 0 0-3 3 3 3 0 0 0 3 3m0 3.54C9.64 9.35 6.5 8 3 8v11c3.5 0 6.64 1.35 9 3.54 2.36-2.19 5.5-3.54 9-3.54V8c-3.5 0-6.64 1.35-9 3.54Z"/></svg>

    </a>
    <label class="md-header__button md-icon" for="__drawer">
      
      <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M3 6h18v2H3V6m0 5h18v2H3v-2m0 5h18v2H3v-2Z"/></svg>
    </label>
    <div class="md-header__title" data-md-component="header-title">
      <div class="md-header__ellipsis">
        <div class="md-header__topic">
          <span class="md-ellipsis">
            Awesome BIDS
          </span>
        </div>
        <div class="md-header__topic" data-md-component="header-topic">
          <span class="md-ellipsis">
            
              Home
            
          </span>
        </div>
      </div>
    </div>
    
      
        <form class="md-header__option" data-md-component="palette">
  
    
    
    
    <input class="md-option" data-md-color-media="(prefers-color-scheme)" data-md-color-scheme="default" data-md-color-primary="indigo" data-md-color-accent="indigo"  aria-label="Switch to light mode"  type="radio" name="__palette" id="__palette_0">
    
      <label class="md-header__button md-icon" title="Switch to light mode" for="__palette_1" hidden>
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="m14.3 16-.7-2h-3.2l-.7 2H7.8L11 7h2l3.2 9h-1.9M20 8.69V4h-4.69L12 .69 8.69 4H4v4.69L.69 12 4 15.31V20h4.69L12 23.31 15.31 20H20v-4.69L23.31 12 20 8.69m-9.15 3.96h2.3L12 9l-1.15 3.65Z"/></svg>
      </label>
    
  
    
    
    
    <input class="md-option" data-md-color-media="(prefers-color-scheme: light)" data-md-color-scheme="default" data-md-color-primary="indigo" data-md-color-accent="indigo"  aria-label="Switch to dark mode"  type="radio" name="__palette" id="__palette_1">
    
      <label class="md-header__button md-icon" title="Switch to dark mode" for="__palette_2" hidden>
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M12 8a4 4 0 0 0-4 4 4 4 0 0 0 4 4 4 4 0 0 0 4-4 4 4 0 0 0-4-4m0 10a6 6 0 0 1-6-6 6 6 0 0 1 6-6 6 6 0 0 1 6 6 6 6 0 0 1-6 6m8-9.31V4h-4.69L12 .69 8.69 4H4v4.69L.69 12 4 15.31V20h4.69L12 23.31 15.31 20H20v-4.69L23.31 12 20 8.69Z"/></svg>
      </label>
    
  
    
    
    
    <input class="md-option" data-md-color-media="(prefers-color-scheme: dark)" data-md-color-scheme="slate" data-md-color-primary="indigo" data-md-color-accent="indigo"  aria-label="Switch to system preference"  type="radio" name="__palette" id="__palette_2">
    
      <label class="md-header__button md-icon" title="Switch to system preference" for="__palette_0" hidden>
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M12 18c-.89 0-1.74-.2-2.5-.55C11.56 16.5 13 14.42 13 12c0-2.42-1.44-4.5-3.5-5.45C10.26 6.2 11.11 6 12 6a6 6 0 0 1 6 6 6 6 0 0 1-6 6m8-9.31V4h-4.69L12 .69 8.69 4H4v4.69L.69 12 4 15.31V20h4.69L12 23.31 15.31 20H20v-4.69L23.31 12 20 8.69Z"/></svg>
      </label>
    
  
</form>
      
    
    
      <script>var media,input,key,value,palette=__md_get("__palette");if(palette&&palette.color){"(prefers-color-scheme)"===palette.color.media&&(media=matchMedia("(prefers-color-scheme: light)"),input=document.querySelector(media.matches?"[data-md-color-media='(prefers-color-scheme: light)']":"[data-md-color-media='(prefers-color-scheme: dark)']"),palette.color.media=input.getAttribute("data-md-color-media"),palette.color.scheme=input.getAttribute("data-md-color-scheme"),palette.color.primary=input.getAttribute("data-md-color-primary"),palette.color.accent=input.getAttribute("data-md-color-accent"));for([key,value]of Object.entries(palette.color))document.body.setAttribute("data-md-color-"+key,value)}</script>
    
    
    
      <label class="md-header__button md-icon" for="__search">
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M9.5 3A6.5 6.5 0 0 1 16 9.5c0 1.61-.59 3.09-1.56 4.23l.27.27h.79l5 5-1.5 1.5-5-5v-.79l-.27-.27A6.516 6.516 0 0 1 9.5 16 6.5 6.5 0 0 1 3 9.5 6.5 6.5 0 0 1 9.5 3m0 2C7 5 5 7 5 9.5S7 14 9.5 14 14 12 14 9.5 12 5 9.5 5Z"/></svg>
      </label>
      <div class="md-search" data-md-component="search" role="dialog">
  <label class="md-search__overlay" for="__search"></label>
  <div class="md-search__inner" role="search">
    <form class="md-search__form" name="search">
      <input type="text" class="md-search__input" name="query" aria-label="Search" placeholder="Search" autocapitalize="off" autocorrect="off" autocomplete="off" spellcheck="false" data-md-component="search-query" required>
      <label class="md-search__icon md-icon" for="__search">
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M9.5 3A6.5 6.5 0 0 1 16 9.5c0 1.61-.59 3.09-1.56 4.23l.27.27h.79l5 5-1.5 1.5-5-5v-.79l-.27-.27A6.516 6.516 0 0 1 9.5 16 6.5 6.5 0 0 1 3 9.5 6.5 6.5 0 0 1 9.5 3m0 2C7 5 5 7 5 9.5S7 14 9.5 14 14 12 14 9.5 12 5 9.5 5Z"/></svg>
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M20 11v2H8l5.5 5.5-1.42 1.42L4.16 12l7.92-7.92L13.5 5.5 8 11h12Z"/></svg>
      </label>
      <nav class="md-search__options" aria-label="Search">
        
        <button type="reset" class="md-search__icon md-icon" title="Clear" aria-label="Clear" tabindex="-1">
          
          <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M19 6.41 17.59 5 12 10.59 6.41 5 5 6.41 10.59 12 5 17.59 6.41 19 12 13.41 17.59 19 19 17.59 13.41 12 19 6.41Z"/></svg>
        </button>
      </nav>
      
    </form>
    <div class="md-search__output">
      <div class="md-search__scrollwrap" data-md-scrollfix>
        <div class="md-search-result" data-md-component="search-result">
          <div class="md-search-result__meta">
            Initializing search
          </div>
          <ol class="md-search-result__list" role="presentation"></ol>
        </div>
      </div>
    </div>
  </div>
</div>
    
    
      <div class="md-header__source">
        <a href="https://github.com/bids-standard/awesome-bids.git" title="Go to repository" class="md-source" data-md-component="source">
  <div class="md-source__icon md-icon">
    
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 496 512"><!--! Font Awesome Free 6.5.1 by @fontawesome - https://fontawesome.com License - https://fontawesome.com/license/free (Icons: CC BY 4.0, Fonts: SIL OFL 1.1, Code: MIT License) Copyright 2023 Fonticons, Inc.--><path d="M165.9 397.4c0 2-2.3 3.6-5.2 3.6-3.3.3-5.6-1.3-5.6-3.6 0-2 2.3-3.6 5.2-3.6 3-.3 5.6 1.3 5.6 3.6zm-31.1-4.5c-.7 2 1.3 4.3 4.3 4.9 2.6 1 5.6 0 6.2-2s-1.3-4.3-4.3-5.2c-2.6-.7-5.5.3-6.2 2.3zm44.2-1.7c-2.9.7-4.9 2.6-4.6 4.9.3 2 2.9 3.3 5.9 2.6 2.9-.7 4.9-2.6 4.6-4.6-.3-1.9-3-3.2-5.9-2.9zM244.8 8C106.1 8 0 113.3 0 252c0 110.9 69.8 205.8 169.5 239.2 12.8 2.3 17.3-5.6 17.3-12.1 0-6.2-.3-40.4-.3-61.4 0 0-70 15-84.7-29.8 0 0-11.4-29.1-27.8-36.6 0 0-22.9-15.7 1.6-15.4 0 0 24.9 2 38.6 25.8 21.9 38.6 58.6 27.5 72.9 20.9 2.3-16 8.8-27.1 16-33.7-55.9-6.2-112.3-14.3-112.3-110.5 0-27.5 7.6-41.3 23.6-58.9-2.6-6.5-11.1-33.3 2.6-67.9 20.9-6.5 69 27 69 27 20-5.6 41.5-8.5 62.8-8.5s42.8 2.9 62.8 8.5c0 0 48.1-33.6 69-27 13.7 34.7 5.2 61.4 2.6 67.9 16 17.7 25.8 31.5 25.8 58.9 0 96.5-58.9 104.2-114.8 110.5 9.2 7.9 17 22.9 17 46.4 0 33.7-.3 75.4-.3 83.6 0 6.5 4.6 14.4 17.3 12.1C428.2 457.8 496 362.9 496 252 496 113.3 383.5 8 244.8 8zM97.2 352.9c-1.3 1-1 3.3.7 5.2 1.6 1.6 3.9 2.3 5.2 1 1.3-1 1-3.3-.7-5.2-1.6-1.6-3.9-2.3-5.2-1zm-10.8-8.1c-.7 1.3.3 2.9 2.3 3.9 1.6 1 3.6.7 4.3-.7.7-1.3-.3-2.9-2.3-3.9-2-.6-3.6-.3-4.3.7zm32.4 35.6c-1.6 1.3-1 4.3 1.3 6.2 2.3 2.3 5.2 2.6 6.5 1 1.3-1.3.7-4.3-1.3-6.2-2.2-2.3-5.2-2.6-6.5-1zm-11.4-14.7c-1.6 1-1.6 3.6 0 5.9 1.6 2.3 4.3 3.3 5.6 2.3 1.6-1.3 1.6-3.9 0-6.2-1.4-2.3-4-3.3-5.6-2z"/></svg>
  </div>
  <div class="md-source__repository">
    GitHub
  </div>
</a>
      </div>
    
  </nav>
  
</header>
    
    <div class="md-container" data-md-component="container">
      
      
        
          
        
      
      <main class="md-main" data-md-component="main">
        <div class="md-main__inner md-grid">
          
            
              
              <div class="md-sidebar md-sidebar--primary" data-md-component="sidebar" data-md-type="navigation" >
                <div class="md-sidebar__scrollwrap">
                  <div class="md-sidebar__inner">
                    



<nav class="md-nav md-nav--primary" aria-label="Navigation" data-md-level="0">
  <label class="md-nav__title" for="__drawer">
    <a href="index.html" title="Awesome BIDS" class="md-nav__button md-logo" aria-label="Awesome BIDS" data-md-component="logo">
      
  
  <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M12 8a3 3 0 0 0 3-3 3 3 0 0 0-3-3 3 3 0 0 0-3 3 3 3 0 0 0 3 3m0 3.54C9.64 9.35 6.5 8 3 8v11c3.5 0 6.64 1.35 9 3.54 2.36-2.19 5.5-3.54 9-3.54V8c-3.5 0-6.64 1.35-9 3.54Z"/></svg>

    </a>
    Awesome BIDS
  </label>
  
    <div class="md-nav__source">
      <a href="https://github.com/bids-standard/awesome-bids.git" title="Go to repository" class="md-source" data-md-component="source">
  <div class="md-source__icon md-icon">
    
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 496 512"><!--! Font Awesome Free 6.5.1 by @fontawesome - https://fontawesome.com License - https://fontawesome.com/license/free (Icons: CC BY 4.0, Fonts: SIL OFL 1.1, Code: MIT License) Copyright 2023 Fonticons, Inc.--><path d="M165.9 397.4c0 2-2.3 3.6-5.2 3.6-3.3.3-5.6-1.3-5.6-3.6 0-2 2.3-3.6 5.2-3.6 3-.3 5.6 1.3 5.6 3.6zm-31.1-4.5c-.7 2 1.3 4.3 4.3 4.9 2.6 1 5.6 0 6.2-2s-1.3-4.3-4.3-5.2c-2.6-.7-5.5.3-6.2 2.3zm44.2-1.7c-2.9.7-4.9 2.6-4.6 4.9.3 2 2.9 3.3 5.9 2.6 2.9-.7 4.9-2.6 4.6-4.6-.3-1.9-3-3.2-5.9-2.9zM244.8 8C106.1 8 0 113.3 0 252c0 110.9 69.8 205.8 169.5 239.2 12.8 2.3 17.3-5.6 17.3-12.1 0-6.2-.3-40.4-.3-61.4 0 0-70 15-84.7-29.8 0 0-11.4-29.1-27.8-36.6 0 0-22.9-15.7 1.6-15.4 0 0 24.9 2 38.6 25.8 21.9 38.6 58.6 27.5 72.9 20.9 2.3-16 8.8-27.1 16-33.7-55.9-6.2-112.3-14.3-112.3-110.5 0-27.5 7.6-41.3 23.6-58.9-2.6-6.5-11.1-33.3 2.6-67.9 20.9-6.5 69 27 69 27 20-5.6 41.5-8.5 62.8-8.5s42.8 2.9 62.8 8.5c0 0 48.1-33.6 69-27 13.7 34.7 5.2 61.4 2.6 67.9 16 17.7 25.8 31.5 25.8 58.9 0 96.5-58.9 104.2-114.8 110.5 9.2 7.9 17 22.9 17 46.4 0 33.7-.3 75.4-.3 83.6 0 6.5 4.6 14.4 17.3 12.1C428.2 457.8 496 362.9 496 252 496 113.3 383.5 8 244.8 8zM97.2 352.9c-1.3 1-1 3.3.7 5.2 1.6 1.6 3.9 2.3 5.2 1 1.3-1 1-3.3-.7-5.2-1.6-1.6-3.9-2.3-5.2-1zm-10.8-8.1c-.7 1.3.3 2.9 2.3 3.9 1.6 1 3.6.7 4.3-.7.7-1.3-.3-2.9-2.3-3.9-2-.6-3.6-.3-4.3.7zm32.4 35.6c-1.6 1.3-1 4.3 1.3 6.2 2.3 2.3 5.2 2.6 6.5 1 1.3-1.3.7-4.3-1.3-6.2-2.2-2.3-5.2-2.6-6.5-1zm-11.4-14.7c-1.6 1-1.6 3.6 0 5.9 1.6 2.3 4.3 3.3 5.6 2.3 1.6-1.3 1.6-3.9 0-6.2-1.4-2.3-4-3.3-5.6-2z"/></svg>
  </div>
  <div class="md-source__repository">
    GitHub
  </div>
</a>
    </div>
  
  <ul class="md-nav__list" data-md-scrollfix>
    
      
      
  
  
    
  
  
  
    <li class="md-nav__item md-nav__item--active">
      
      <input class="md-nav__toggle md-toggle" type="checkbox" id="__toc">
      
      
        
      
      
        <label class="md-nav__link md-nav__link--active" for="__toc">
          
  
  <span class="md-ellipsis">
    Home
  </span>
  

          <span class="md-nav__icon md-icon"></span>
        </label>
      
      <a href="index.html" class="md-nav__link md-nav__link--active">
        
  
  <span class="md-ellipsis">
    Home
  </span>
  

      </a>
      
        

<nav class="md-nav md-nav--secondary" aria-label="Table of contents">
  
  
  
    
  
  
    <label class="md-nav__title" for="__toc">
      <span class="md-nav__icon md-icon"></span>
      Table of contents
    </label>
    <ul class="md-nav__list" data-md-component="toc" data-md-scrollfix>
      
        <li class="md-nav__item">
  <a href="#documentation" class="md-nav__link">
    <span class="md-ellipsis">
      Documentation
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#community" class="md-nav__link">
    <span class="md-ellipsis">
      Community
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#social-media" class="md-nav__link">
    <span class="md-ellipsis">
      Social media
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#datasets" class="md-nav__link">
    <span class="md-ellipsis">
      Datasets
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#converters" class="md-nav__link">
    <span class="md-ellipsis">
      Converters
    </span>
  </a>
  
    <nav class="md-nav" aria-label="Converters">
      <ul class="md-nav__list">
        
          <li class="md-nav__item">
  <a href="#asl" class="md-nav__link">
    <span class="md-ellipsis">
      ASL
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#bids" class="md-nav__link">
    <span class="md-ellipsis">
      BIDS
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#eeg" class="md-nav__link">
    <span class="md-ellipsis">
      EEG
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#meg" class="md-nav__link">
    <span class="md-ellipsis">
      MEG
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#misc" class="md-nav__link">
    <span class="md-ellipsis">
      MISC
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#mri" class="md-nav__link">
    <span class="md-ellipsis">
      MRI
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#nirs" class="md-nav__link">
    <span class="md-ellipsis">
      NIRS
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#pet" class="md-nav__link">
    <span class="md-ellipsis">
      PET
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#tsv" class="md-nav__link">
    <span class="md-ellipsis">
      TSV
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#behavioral" class="md-nav__link">
    <span class="md-ellipsis">
      behavioral
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#computational-model" class="md-nav__link">
    <span class="md-ellipsis">
      computational model
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#events" class="md-nav__link">
    <span class="md-ellipsis">
      events
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#ieeg" class="md-nav__link">
    <span class="md-ellipsis">
      iEEG
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#physiological" class="md-nav__link">
    <span class="md-ellipsis">
      physiological
    </span>
  </a>
  
</li>
        
      </ul>
    </nav>
  
</li>
      
        <li class="md-nav__item">
  <a href="#validation" class="md-nav__link">
    <span class="md-ellipsis">
      Validation
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#bids-apps" class="md-nav__link">
    <span class="md-ellipsis">
      BIDS Apps
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#tools" class="md-nav__link">
    <span class="md-ellipsis">
      Tools
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#code" class="md-nav__link">
    <span class="md-ellipsis">
      Code
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#bids-extension-proposals-bep" class="md-nav__link">
    <span class="md-ellipsis">
      BIDS Extension Proposals (BEP)
    </span>
  </a>
  
    <nav class="md-nav" aria-label="BIDS Extension Proposals (BEP)">
      <ul class="md-nav__list">
        
          <li class="md-nav__item">
  <a href="#raw" class="md-nav__link">
    <span class="md-ellipsis">
      raw
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#derivative" class="md-nav__link">
    <span class="md-ellipsis">
      derivative
    </span>
  </a>
  
</li>
        
      </ul>
    </nav>
  
</li>
      
        <li class="md-nav__item">
  <a href="#publications" class="md-nav__link">
    <span class="md-ellipsis">
      Publications
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#contribute" class="md-nav__link">
    <span class="md-ellipsis">
      Contribute
    </span>
  </a>
  
</li>
      
    </ul>
  
</nav>
      
    </li>
  

    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href="contributing.html" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Contributing
  </span>
  

      </a>
    </li>
  

    
  </ul>
</nav>
                  </div>
                </div>
              </div>
            
            
              
              <div class="md-sidebar md-sidebar--secondary" data-md-component="sidebar" data-md-type="toc" >
                <div class="md-sidebar__scrollwrap">
                  <div class="md-sidebar__inner">
                    

<nav class="md-nav md-nav--secondary" aria-label="Table of contents">
  
  
  
    
  
  
    <label class="md-nav__title" for="__toc">
      <span class="md-nav__icon md-icon"></span>
      Table of contents
    </label>
    <ul class="md-nav__list" data-md-component="toc" data-md-scrollfix>
      
        <li class="md-nav__item">
  <a href="#documentation" class="md-nav__link">
    <span class="md-ellipsis">
      Documentation
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#community" class="md-nav__link">
    <span class="md-ellipsis">
      Community
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#social-media" class="md-nav__link">
    <span class="md-ellipsis">
      Social media
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#datasets" class="md-nav__link">
    <span class="md-ellipsis">
      Datasets
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#converters" class="md-nav__link">
    <span class="md-ellipsis">
      Converters
    </span>
  </a>
  
    <nav class="md-nav" aria-label="Converters">
      <ul class="md-nav__list">
        
          <li class="md-nav__item">
  <a href="#asl" class="md-nav__link">
    <span class="md-ellipsis">
      ASL
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#bids" class="md-nav__link">
    <span class="md-ellipsis">
      BIDS
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#eeg" class="md-nav__link">
    <span class="md-ellipsis">
      EEG
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#meg" class="md-nav__link">
    <span class="md-ellipsis">
      MEG
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#misc" class="md-nav__link">
    <span class="md-ellipsis">
      MISC
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#mri" class="md-nav__link">
    <span class="md-ellipsis">
      MRI
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#nirs" class="md-nav__link">
    <span class="md-ellipsis">
      NIRS
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#pet" class="md-nav__link">
    <span class="md-ellipsis">
      PET
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#tsv" class="md-nav__link">
    <span class="md-ellipsis">
      TSV
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#behavioral" class="md-nav__link">
    <span class="md-ellipsis">
      behavioral
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#computational-model" class="md-nav__link">
    <span class="md-ellipsis">
      computational model
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#events" class="md-nav__link">
    <span class="md-ellipsis">
      events
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#ieeg" class="md-nav__link">
    <span class="md-ellipsis">
      iEEG
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#physiological" class="md-nav__link">
    <span class="md-ellipsis">
      physiological
    </span>
  </a>
  
</li>
        
      </ul>
    </nav>
  
</li>
      
        <li class="md-nav__item">
  <a href="#validation" class="md-nav__link">
    <span class="md-ellipsis">
      Validation
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#bids-apps" class="md-nav__link">
    <span class="md-ellipsis">
      BIDS Apps
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#tools" class="md-nav__link">
    <span class="md-ellipsis">
      Tools
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#code" class="md-nav__link">
    <span class="md-ellipsis">
      Code
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#bids-extension-proposals-bep" class="md-nav__link">
    <span class="md-ellipsis">
      BIDS Extension Proposals (BEP)
    </span>
  </a>
  
    <nav class="md-nav" aria-label="BIDS Extension Proposals (BEP)">
      <ul class="md-nav__list">
        
          <li class="md-nav__item">
  <a href="#raw" class="md-nav__link">
    <span class="md-ellipsis">
      raw
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#derivative" class="md-nav__link">
    <span class="md-ellipsis">
      derivative
    </span>
  </a>
  
</li>
        
      </ul>
    </nav>
  
</li>
      
        <li class="md-nav__item">
  <a href="#publications" class="md-nav__link">
    <span class="md-ellipsis">
      Publications
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#contribute" class="md-nav__link">
    <span class="md-ellipsis">
      Contribute
    </span>
  </a>
  
</li>
      
    </ul>
  
</nav>
                  </div>
                </div>
              </div>
            
          
          
            <div class="md-content" data-md-component="content">
              <article class="md-content__inner md-typeset">
                
                  

  
    <a href="https://github.com/bids-standard/awesome-bids.git/edit/main/src/index.md" title="Edit this page" class="md-content__button md-icon">
      
      <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M20.71 7.04c.39-.39.39-1.04 0-1.41l-2.34-2.34c-.37-.39-1.02-.39-1.41 0l-1.84 1.83 3.75 3.75M3 17.25V21h3.75L17.81 9.93l-3.75-3.75L3 17.25Z"/></svg>
    </a>
  
  


<h1 id="awesome-bids"><a class="toclink" href="#awesome-bids">Awesome BIDS <a href="https://github.com/sindresorhus/awesome#readme"><img alt="Awesome" src="https://awesome.re/badge.svg" /></a></a></h1>
<p><img src="https://raw.githubusercontent.com/bids-standard/bids-specification/master/BIDS_logo/BIDS_logo_white_transparent_background_crop.png#gh-dark-mode-only" alt="bids-logo" width="600"/>
<img src="https://raw.githubusercontent.com/bids-standard/bids-specification/master/BIDS_logo//BIDS_logo_black_transparent_background_crop.png#gh-light-mode-only" alt="bids-logo" width="600"/></p>
<p>A curated list of awesome projects, proposals, apps and resources
related to the Brain Imaging Data Structure.</p>
<h2 id="documentation"><a class="toclink" href="#documentation">Documentation</a></h2>
<p>Documentation related to BIDS.</p>
<ul>
<li>The central <a href="http://www.bids-standard.org/">BIDS website</a> to get general information about BIDS.</li>
<li>The <a href="https://bids-website.readthedocs.io/en/latest/getting_started/index.html">BIDS Starter Kit</a>
  is the best place to get started with BIDS.</li>
<li>The official <a href="https://bids-specification.readthedocs.io">BIDS Specification</a> to get into the details.</li>
<li>The <a href="https://www.youtube.com/channel/UCxZUcYfd_nvIVWAbzRB1tlw">BIDS youtube channel</a>
  where we try to curate <a href="https://www.youtube.com/@brainimagingdatastructure6721/playlists">playlists</a> of BIDS related videos.</li>
<li>Slides of many BIDS related presentations are available on the <a href="https://osf.io/yn93h/">Open Science Framework</a>.</li>
<li>The <a href="https://www.fieldtriptoolbox.org/example/bids/">FieldTrip</a> websites contains
  many BIDS tips and  examples, mainly for MEG, EEG, fNIRS, etc.</li>
</ul>
<h2 id="community"><a class="toclink" href="#community">Community</a></h2>
<ul>
<li>
<p>The <a href="https://neurostars.org/tag/bids">Neurostars discourse forum</a>
  run by the International Neuroinformatics Coordinating Facility (<a href="https://www.incf.org/">INCF</a>)
  with its own <a href="https://neurostars.org/tag/bids">BIDS category</a>.
  With a lot BIDS users and developers,
  it is the best place where to ask BIDS related questions.</p>
</li>
<li>
<p><a href="https://groups.google.com/g/bids-discussion">BIDS mailing list</a>:
  a google group for announcements and discussions around BIDS.</p>
</li>
<li>
<p><a href="https://mattermost.brainhack.org/brainhack/channels/bids_general">BIDS in the Brainhack mattersmost</a>:
  <a href="https://mattermost.brainhack.org">mattermost</a> is the open source equivalent of slack
  and the <a href="https://brainhack.org/">Brainhack</a> instance has over 5000 members
  with its own channel dedicated BIDS channel.</p>
</li>
</ul>
<h2 id="social-media"><a class="toclink" href="#social-media">Social media</a></h2>
<p>Here are the official BIDS account on several social media.</p>
<ul>
<li><a href="https://fosstodon.org/@bidsstandard/">Mastodon</a></li>
<li><a href="https://bsky.app/profile/bidsstandard.bsky.social">Bluesky</a></li>
<li><a href="https://www.instagram.com/bidsstandard/">Instagram</a></li>
<li><a href="https://www.youtube.com/channel/UCxZUcYfd_nvIVWAbzRB1tlw">YouTube</a></li>
<li>Our podcast on <a href="https://anchor.fm/bids-maintenance">anchor</a></li>
</ul>
<h2 id="datasets"><a class="toclink" href="#datasets">Datasets</a></h2>
<ul>
<li>The <a href="https://bids-standard.github.io/bids-examples/">BIDS examples</a> repository hosts dataset of each modality
  with empty raw data files. These datasets can be useful to:</li>
<li>serve as an example on how a BIDS dataset can be structured</li>
<li>write lightweight software tests</li>
<li><a href="https://openneuro.org/">OpenNeuro</a> host more than 1000 open BIDS datasets of all datatypes.</li>
</ul>
<h2 id="converters"><a class="toclink" href="#converters">Converters</a></h2>
<p>Tools for converting data to/from BIDS from other standard or custom formats and layouts.</p>
<!-- Converters starts -->

<h3 id="asl"><a class="toclink" href="#asl">ASL</a></h3>
<ul>
<li><img src='./images/logo_matlab.png' width='17px'>  <a href="https://exploreasl.github.io/Documentation/1.8.0/Tutorials-ASL-BIDS/">Explore ASL</a>: Convert DICOM and NIFTI data to the ASL-BIDS format.
  <br><a href="https://github.com/ExploreASL/ExploreASL"><img alt="Last commit" src="https://img.shields.io/github/last-commit/ExploreASL/ExploreASL?style=plastic" /></a></li>
<li><img src='./images/logo_python.png' width='14px'>  <a href="https://brainlife.io/docs/using_ezBIDS/">ezBIDS</a>: A web-based BIDS conversion tool with four unique features: (1) No installation or programming requirements. (2) Handling of both imaging and task events data and metadata. (3) Semi-automated inference and guidance for adherence to BIDS. (4) Multiple data management options, including download BIDS data to local system, or transfer to OpenNeuro.org or to brainlife.io.
  <br><img alt="" src="https://img.shields.io/badge/Javascript-yellow.svg?style=plastic" /><img alt="" src="https://img.shields.io/badge/shell-black.svg?style=plastic" /><a href="https://github.com/brainlife/ezbids"><img alt="Last commit" src="https://img.shields.io/github/last-commit/brainlife/ezbids?style=plastic" /></a></li>
</ul>
<h3 id="bids"><a class="toclink" href="#bids">BIDS</a></h3>
<ul>
<li><img src='./images/logo_python.png' width='14px'>  <a href="https://github.com/bids-standard/BIDS2ISATab">BIDS2ISATab</a>: Extract ISA-Tab compatible metadata from BIDS
  <br><a href="https://github.com/bids-standard/BIDS2ISATab"><img alt="Last commit" src="https://img.shields.io/github/last-commit/bids-standard/BIDS2ISATab?style=plastic" /></a></li>
<li><img src='./images/logo_python.png' width='14px'>  <a href="https://github.com/bids-standard/BIDS2NDA">BIDS2NDA</a>: Extract NIHM Data Archive compatible metadata from BIDS
  <br><a href="https://github.com/bids-standard/BIDS2NDA"><img alt="Last commit" src="https://img.shields.io/github/last-commit/bids-standard/BIDS2NDA?style=plastic" /></a></li>
<li><img src='./images/logo_python.png' width='14px'>  <a href="https://github.com/lwallace23/bids2xar">bids2xar - for XNAT import</a>: Convert BIDS data set into XNAT XAR bundles
  <br><a href="https://github.com/lwallace23/bids2xar"><img alt="Last commit" src="https://img.shields.io/github/last-commit/lwallace23/bids2xar?style=plastic" /></a></li>
</ul>
<h3 id="eeg"><a class="toclink" href="#eeg">EEG</a></h3>
<ul>
<li><img src='./images/logo_python.png' width='14px'>  <a href="https://gitlab.uliege.be/CyclotronResearchCentre/Public/bidstools/bidsme/bidsme_example/-/tree/master/example1">BIDSme</a>:
  <br><a href="https://github.com/nbeliy/bidsme"><img alt="Last commit" src="https://img.shields.io/github/last-commit/nbeliy/bidsme?style=plastic" /></a></li>
<li><img src='./images/logo_python.png' width='14px'>  <a href="https://github.com/aces/EEG2BIDS">EEG2BIDS</a>: A tool for converting raw EEG and iEEG data into the BIDS standard data structure, prepared for LORIS (Longitudinal Online Research and Imaging System).
  <br><img alt="" src="https://img.shields.io/badge/Javascript-yellow.svg?style=plastic" /><a href="https://github.com/aces/EEG2BIDS"><img alt="Last commit" src="https://img.shields.io/github/last-commit/aces/EEG2BIDS?style=plastic" /></a></li>
<li><img src='./images/logo_matlab.png' width='17px'>  <a href="https://eeglab.org/tutorials/04_Import/BIDS.html">EEGLAB</a>: <a href='https://github.com/arnodelorme/bids-MATLAB-tools'>See plugins</a>
  <br><a href="https://github.com/sccn/eeglab"><img alt="Last commit" src="https://img.shields.io/github/last-commit/sccn/eeglab?style=plastic" /></a></li>
<li><img src='./images/logo_matlab.png' width='17px'>  <a href="https://www.fieldtriptoolbox.org/example/bids/">FieldTrip - data2bids</a>:
  <br><a href="https://github.com/fieldtrip/fieldtrip"><img alt="Last commit" src="https://img.shields.io/github/last-commit/fieldtrip/fieldtrip?style=plastic" /></a></li>
<li><img src='./images/logo_python.png' width='14px'>  <a href="https://mne.tools/mne-bids">MNE-BIDS</a>: MNE-BIDS is a Python package that allows you to read and write BIDS-compatible datasets with the help of MNE-Python.
  <br><a href="https://github.com/mne-tools/mne-bids"><img alt="Last commit" src="https://img.shields.io/github/last-commit/mne-tools/mne-bids?style=plastic" /></a><a href="https://pypi.org/project/mne-bids"><img alt="PyPI version" src="https://badge.fury.io/py/mne-bids.svg" /></a></li>
<li><img src='./images/logo_python.png' width='14px'>  <a href="https://sovabids.readthedocs.io/en/latest/">sovabids</a>: A Python package for the automatic conversion of EEG datasets to the BIDS standard, with a focus on making the most out of metadata.
  <br><a href="https://github.com/yjmantilla/sovabids"><img alt="Last commit" src="https://img.shields.io/github/last-commit/yjmantilla/sovabids?style=plastic" /></a></li>
</ul>
<h3 id="meg"><a class="toclink" href="#meg">MEG</a></h3>
<ul>
<li><img src='./images/logo_python.png' width='14px'>  <a href="https://macquarie-meg-research.github.io/Biscuit/">Biscuit</a>: GUI for easy MEG to BIDS conversion
  <br><a href="https://github.com/Macquarie-MEG-Research/Biscuit"><img alt="Last commit" src="https://img.shields.io/github/last-commit/Macquarie-MEG-Research/Biscuit?style=plastic" /></a></li>
<li><img src='./images/logo_matlab.png' width='17px'>  <a href="https://eeglab.org/tutorials/04_Import/BIDS.html">EEGLAB</a>: <a href='https://github.com/arnodelorme/bids-MATLAB-tools'>See plugins</a>
  <br><a href="https://github.com/sccn/eeglab"><img alt="Last commit" src="https://img.shields.io/github/last-commit/sccn/eeglab?style=plastic" /></a></li>
<li><img src='./images/logo_matlab.png' width='17px'>  <a href="https://www.fieldtriptoolbox.org/example/bids/">FieldTrip - data2bids</a>:
  <br><a href="https://github.com/fieldtrip/fieldtrip"><img alt="Last commit" src="https://img.shields.io/github/last-commit/fieldtrip/fieldtrip?style=plastic" /></a></li>
<li><img src='./images/logo_python.png' width='14px'>  <a href="https://mne.tools/mne-bids">MNE-BIDS</a>: MNE-BIDS is a Python package that allows you to read and write BIDS-compatible datasets with the help of MNE-Python.
  <br><a href="https://github.com/mne-tools/mne-bids"><img alt="Last commit" src="https://img.shields.io/github/last-commit/mne-tools/mne-bids?style=plastic" /></a><a href="https://pypi.org/project/mne-bids"><img alt="PyPI version" src="https://badge.fury.io/py/mne-bids.svg" /></a></li>
</ul>
<h3 id="misc"><a class="toclink" href="#misc">MISC</a></h3>
<ul>
<li><img src='./images/logo_python.png' width='14px'>  <a href="https://github.com/tsalo/convert-eprime">convert-eprime</a>: Python functions to convert E-Prime files to CSVs. Not currently being developed.
  <br><a href="https://github.com/tsalo/convert-eprime"><img alt="Last commit" src="https://img.shields.io/github/last-commit/tsalo/convert-eprime?style=plastic" /></a></li>
<li><img src='./images/logo_python.png' width='14px'>  <a href="https://github.com/dissagaliyeva/sim2bids">sim2bids</a>: GUI to easily convert simulation results to BIDS format, according to <a href="https://bids.neuroimaging.io/bep034" target="_blank"> BEP 34 </a>.
  <br><a href="https://github.com/dissagaliyeva/sim2bids"><img alt="Last commit" src="https://img.shields.io/github/last-commit/dissagaliyeva/sim2bids?style=plastic" /></a></li>
</ul>
<h3 id="mri"><a class="toclink" href="#mri">MRI</a></h3>
<ul>
<li><img src='./images/logo_python.png' width='14px'>  <a href="https://bidscoin.readthedocs.io/en/stable/">BIDScoin</a>: BIDScoin is a flexible tool to convert (“coin”) source-level (raw) neuroimaging data sets to BIDS without needing to code anything. It features automatic data discovery based on header as well as filesystem information, and comes with a user-friendly GUI to add missing information and tweak the results. BIDScoin supports multiple source data formats with plugins (e.g. employing dcm2niix, spec2nii or nibabel) and allows customization of the prior knowledge about your data (allowing for fully automatic CLI data conversion if you like).
  <br><a href="https://github.com/Donders-Institute/bidscoin"><img alt="Last commit" src="https://img.shields.io/github/last-commit/Donders-Institute/bidscoin?style=plastic" /></a><a href="https://pypi.org/project/bidscoin"><img alt="PyPI version" src="https://badge.fury.io/py/bidscoin.svg" /></a></li>
<li><img src='./images/logo_R.png' width='18px'>  <a href="https://bidsconvertr.github.io/">BIDSconvertR</a>: The BIDSconvertR R package provides a user-friendly workflow with graphical user interfaces. It consists of the following steps: (i) convert DICOM data to NIfTI data using dcm2niix (ii) structure this data according to the BIDS specification (iii) provide the papayaWidget viewer for inspecting the images
  <br><a href="https://github.com/bidsconvertr/bidsconvertr"><img alt="Last commit" src="https://img.shields.io/github/last-commit/bidsconvertr/bidsconvertr?style=plastic" /></a></li>
<li><img src='./images/logo_python.png' width='14px'>  <a href="https://github.com/jmtyszka/bidskit/blob/master/docs/QuickStart.md">bidskit</a>: Utility functions for working with DICOM and BIDS neuroimaging data.
  <br><a href="https://github.com/jmtyszka/bidskit"><img alt="Last commit" src="https://img.shields.io/github/last-commit/jmtyszka/bidskit?style=plastic" /></a><a href="https://pypi.org/project/bidskit"><img alt="PyPI version" src="https://badge.fury.io/py/bidskit.svg" /></a><a href="https://hub.docker.com/r/jmtyszka/bidskit"><img alt="Docker version" src="https://img.shields.io/docker/pulls/jmtyszka/bidskit.svg?style=plastic" /></a></li>
<li><img src='./images/logo_python.png' width='14px'>  <a href="https://gitlab.uliege.be/CyclotronResearchCentre/Public/bidstools/bidsme/bidsme_example/-/tree/master/example1">BIDSme</a>:
  <br><a href="https://github.com/nbeliy/bidsme"><img alt="Last commit" src="https://img.shields.io/github/last-commit/nbeliy/bidsme?style=plastic" /></a></li>
<li><img src='./images/logo_python.png' width='14px'>  <a href="https://github.com/ColinVDB/BMAT">BMAT</a>:
  <br><a href="https://github.com/ColinVDB/BMAT"><img alt="Last commit" src="https://img.shields.io/github/last-commit/ColinVDB/BMAT?style=plastic" /></a><a href="https://hub.docker.com/r/colinvdb/bmat-dcm2niix"><img alt="Docker version" src="https://img.shields.io/docker/pulls/colinvdb/bmat-dcm2niix.svg?style=plastic" /></a></li>
<li><img src='./images/logo_python.png' width='14px'>  <a href="https://github.com/BrkRaw/brkraw">BrkRaw</a>: For a preclinical Bruker MRI scanner
  <br><a href="https://github.com/BrkRaw/brkraw"><img alt="Last commit" src="https://img.shields.io/github/last-commit/BrkRaw/brkraw?style=plastic" /></a></li>
<li><img src='./images/logo_python.png' width='14px'>  <a href="https://aramislab.paris.inria.fr/clinica/docs/public/dev/">Clinica</a>:
  <br><a href="https://github.com/aramis-lab/clinica"><img alt="Last commit" src="https://img.shields.io/github/last-commit/aramis-lab/clinica?style=plastic" /></a></li>
<li><img src='./images/logo_python.png' width='14px'>  <a href="https://github.com/dangom/dac2bids">dac2bids</a>: Create a BIDS structure for a DICOM folder.
  <br><a href="https://github.com/dangom/dac2bids"><img alt="Last commit" src="https://img.shields.io/github/last-commit/dangom/dac2bids?style=plastic" /></a></li>
<li><img src='./images/logo_python.png' width='14px'>  <a href="https://github.com/SIMEXP/Data2Bids">Data2Bids</a>: Converts MRI files from extension supported by nibabel into NIfTI and convert them to BIDS
  <br><a href="https://github.com/SIMEXP/Data2Bids"><img alt="Last commit" src="https://img.shields.io/github/last-commit/SIMEXP/Data2Bids?style=plastic" /></a><a href="https://pypi.org/project/data2bids"><img alt="PyPI version" src="https://badge.fury.io/py/data2bids.svg" /></a></li>
<li><img src='./images/logo_python.png' width='14px'>  <a href="https://unfmontreal.github.io/Dcm2Bids/">Dcm2Bids</a>: converts DICOM files using dcm2niix into BIDS
  <br><a href="https://github.com/cbedetti/Dcm2Bids"><img alt="Last commit" src="https://img.shields.io/github/last-commit/cbedetti/Dcm2Bids?style=plastic" /></a><a href="https://pypi.org/project/Dcm2Bids"><img alt="PyPI version" src="https://badge.fury.io/py/Dcm2Bids.svg" /></a><a href="https://hub.docker.com/r/unfmontreal/dcm2bids"><img alt="Docker version" src="https://img.shields.io/docker/pulls/unfmontreal/dcm2bids.svg?style=plastic" /></a></li>
<li><img src='./images/logo_matlab.png' width='17px'>  <a href="https://exploreasl.github.io/Documentation/1.8.0/Tutorials-ASL-BIDS/">Explore ASL</a>: Convert DICOM and NIFTI data to the ASL-BIDS format.
  <br><a href="https://github.com/ExploreASL/ExploreASL"><img alt="Last commit" src="https://img.shields.io/github/last-commit/ExploreASL/ExploreASL?style=plastic" /></a></li>
<li><img src='./images/logo_python.png' width='14px'>  <a href="https://brainlife.io/docs/using_ezBIDS/">ezBIDS</a>: A web-based BIDS conversion tool with four unique features: (1) No installation or programming requirements. (2) Handling of both imaging and task events data and metadata. (3) Semi-automated inference and guidance for adherence to BIDS. (4) Multiple data management options, including download BIDS data to local system, or transfer to OpenNeuro.org or to brainlife.io.
  <br><img alt="" src="https://img.shields.io/badge/Javascript-yellow.svg?style=plastic" /><img alt="" src="https://img.shields.io/badge/shell-black.svg?style=plastic" /><a href="https://github.com/brainlife/ezbids"><img alt="Last commit" src="https://img.shields.io/github/last-commit/brainlife/ezbids?style=plastic" /></a></li>
<li><img src='./images/logo_matlab.png' width='17px'>  <a href="https://www.fieldtriptoolbox.org/example/bids/">FieldTrip - data2bids</a>:
  <br><a href="https://github.com/fieldtrip/fieldtrip"><img alt="Last commit" src="https://img.shields.io/github/last-commit/fieldtrip/fieldtrip?style=plastic" /></a></li>
<li><img src='./images/logo_python.png' width='14px'>  <a href="https://heudiconv.readthedocs.io/">HeuDiConv</a>: A flexible DICOM converter for organizing brain imaging data into structured directory layouts
  <br><a href="https://github.com/nipy/heudiconv"><img alt="Last commit" src="https://img.shields.io/github/last-commit/nipy/heudiconv?style=plastic" /></a><a href="https://pypi.org/project/heudiconv"><img alt="PyPI version" src="https://badge.fury.io/py/heudiconv.svg" /></a><a href="https://hub.docker.com/r/nipy/heudiconv"><img alt="Docker version" src="https://img.shields.io/docker/pulls/nipy/heudiconv.svg?style=plastic" /></a></li>
<li><a href="https://github.com/mslw/horos-bids-output">Horos (Osirix) export plugin</a>: Horos plugin for BIDS output.
  <br><a href="https://github.com/mslw/horos-bids-output"><img alt="Last commit" src="https://img.shields.io/github/last-commit/mslw/horos-bids-output?style=plastic" /></a></li>
<li><img src='./images/logo_python.png' width='14px'>  <a href="https://github.com/mercure-imaging/mercure-dcm2bids">mercure-dcm2bids</a>: A containerized app that can be used to perform BIDS conversion of DICOM studies sent directly to mercure from a scanner or PACS. mercure is an open-source DICOM orchestration platform that can integrate containerized apps into clinical workflows. It has a graphical user interface making it easy to setup and manage BIDS configurations for multiple protocols. The Dcm2Bids tool is used for conversion.
  <br><a href="https://github.com/mercure-imaging/mercure-dcm2bids"><img alt="Last commit" src="https://img.shields.io/github/last-commit/mercure-imaging/mercure-dcm2bids?style=plastic" /></a><a href="https://hub.docker.com/r/mercureimaging/mercure-dcm2bids"><img alt="Docker version" src="https://img.shields.io/docker/pulls/mercureimaging/mercure-dcm2bids.svg?style=plastic" /></a></li>
<li><img src='./images/logo_python.png' width='14px'>  <a href="https://github.com/benoitberanger/niix2bids">niix2bids</a>: Use this package as a command line to organize your Nifti dataset into BIDS.
  <br><a href="https://github.com/benoitberanger/niix2bids"><img alt="Last commit" src="https://img.shields.io/github/last-commit/benoitberanger/niix2bids?style=plastic" /></a></li>
<li><a href="https://github.com/bids-standard/openfmri2bids">OpenfMRI2BIDS</a>: Convert OpenfMRI dataset to BIDS
  <br><a href="https://github.com/bids-standard/openfmri2bids"><img alt="Last commit" src="https://img.shields.io/github/last-commit/bids-standard/openfmri2bids?style=plastic" /></a></li>
<li><img src='./images/logo_python.png' width='14px'>  <a href="https://github.com/ReproNim/reproin">ReproIn</a>: HeuDiConv-based turnkey solution: a setup for automatic generation of shareable, version-controlled BIDS datasets from MR scanners.
  <br><a href="https://github.com/ReproNim/reproin"><img alt="Last commit" src="https://img.shields.io/github/last-commit/ReproNim/reproin?style=plastic" /></a><a href="https://hub.docker.com/r/repronim/reproin"><img alt="Docker version" src="https://img.shields.io/docker/pulls/repronim/reproin.svg?style=plastic" /></a></li>
<li><img src='./images/logo_python.png' width='14px'>  <a href="https://github.com/IBT-FMI/SAMRI">SAMRI</a>: Full stack Small Animal MRI data analysis package, including the <code>bru2bids</code> repositing pipeline, which can convert Bruker archives to the BIDS format. From the ETH and University of Zurich, with collaboration from MIT and Dartmouth College.
  <br><img alt="" src="https://img.shields.io/badge/shell-black.svg?style=plastic" /><a href="https://github.com/IBT-FMI/SAMRI"><img alt="Last commit" src="https://img.shields.io/github/last-commit/IBT-FMI/SAMRI?style=plastic" /></a></li>
<li><a href="https://github.com/kamillipi/2bids">XNAT2BIDS</a>: Simple xnat pipeline to convert DICOM scans to BIDS-compatible output (nii+json).
  <br><img alt="" src="https://img.shields.io/badge/shell-black.svg?style=plastic" /><a href="https://github.com/kamillipi/2bids"><img alt="Last commit" src="https://img.shields.io/github/last-commit/kamillipi/2bids?style=plastic" /></a></li>
</ul>
<h3 id="nirs"><a class="toclink" href="#nirs">NIRS</a></h3>
<ul>
<li><img src='./images/logo_python.png' width='14px'>  <a href="https://mne.tools/mne-bids">MNE-BIDS</a>: MNE-BIDS is a Python package that allows you to read and write BIDS-compatible datasets with the help of MNE-Python.
  <br><a href="https://github.com/mne-tools/mne-bids"><img alt="Last commit" src="https://img.shields.io/github/last-commit/mne-tools/mne-bids?style=plastic" /></a><a href="https://pypi.org/project/mne-bids"><img alt="PyPI version" src="https://badge.fury.io/py/mne-bids.svg" /></a></li>
</ul>
<h3 id="pet"><a class="toclink" href="#pet">PET</a></h3>
<ul>
<li><img src='./images/logo_python.png' width='14px'>  <a href="https://bidscoin.readthedocs.io/en/stable/">BIDScoin</a>: BIDScoin is a flexible tool to convert (“coin”) source-level (raw) neuroimaging data sets to BIDS without needing to code anything. It features automatic data discovery based on header as well as filesystem information, and comes with a user-friendly GUI to add missing information and tweak the results. BIDScoin supports multiple source data formats with plugins (e.g. employing dcm2niix, spec2nii or nibabel) and allows customization of the prior knowledge about your data (allowing for fully automatic CLI data conversion if you like).
  <br><a href="https://github.com/Donders-Institute/bidscoin"><img alt="Last commit" src="https://img.shields.io/github/last-commit/Donders-Institute/bidscoin?style=plastic" /></a><a href="https://pypi.org/project/bidscoin"><img alt="PyPI version" src="https://badge.fury.io/py/bidscoin.svg" /></a></li>
<li><img src='./images/logo_python.png' width='14px'>  <a href="https://aramislab.paris.inria.fr/clinica/docs/public/dev/">Clinica</a>:
  <br><a href="https://github.com/aramis-lab/clinica"><img alt="Last commit" src="https://img.shields.io/github/last-commit/aramis-lab/clinica?style=plastic" /></a></li>
<li><img src='./images/logo_matlab.png' width='17px'> <img src='./images/logo_python.png' width='14px'>  <a href="https://github.com/openneuropet/PET2BIDS">PET2BIDS</a>: Helps you convert your PET data! raw PET scanner files (for example ecat, dicom) and additional side file like excel sheets.
  <br><a href="https://github.com/openneuropet/PET2BIDS"><img alt="Last commit" src="https://img.shields.io/github/last-commit/openneuropet/PET2BIDS?style=plastic" /></a></li>
</ul>
<h3 id="tsv"><a class="toclink" href="#tsv">TSV</a></h3>
<ul>
<li><img src='./images/logo_python.png' width='14px'>  <a href="https://github.com/bids-standard/bidsutils/tree/master/BIDSto3col">BIDSto3col</a>: Reads BidsTSV and then creates 3 column event files, one per event type if a "trial_type" column is found.
  <br><img alt="" src="https://img.shields.io/badge/shell-black.svg?style=plastic" /><a href="https://github.com/bids-standard/bidsutils/tree/master/BIDSto3col"><img alt="Last commit" src="https://img.shields.io/github/last-commit/bids-standard/bidsutils/tree/master/BIDSto3col?style=plastic" /></a></li>
</ul>
<h3 id="behavioral"><a class="toclink" href="#behavioral">behavioral</a></h3>
<ul>
<li><img src='./images/logo_python.png' width='14px'>  <a href="https://github.com/tsalo/convert-eprime">convert-eprime</a>: Python functions to convert E-Prime files to CSVs. Not currently being developed.
  <br><a href="https://github.com/tsalo/convert-eprime"><img alt="Last commit" src="https://img.shields.io/github/last-commit/tsalo/convert-eprime?style=plastic" /></a></li>
<li><img src='./images/logo_matlab.png' width='17px'>  <a href="https://www.fieldtriptoolbox.org/example/bids/">FieldTrip - data2bids</a>:
  <br><a href="https://github.com/fieldtrip/fieldtrip"><img alt="Last commit" src="https://img.shields.io/github/last-commit/fieldtrip/fieldtrip?style=plastic" /></a></li>
</ul>
<h3 id="computational-model"><a class="toclink" href="#computational-model">computational model</a></h3>
<ul>
<li><img src='./images/logo_python.png' width='14px'>  <a href="https://github.com/dissagaliyeva/sim2bids">sim2bids</a>: GUI to easily convert simulation results to BIDS format, according to <a href="https://bids.neuroimaging.io/bep034" target="_blank"> BEP 34 </a>.
  <br><a href="https://github.com/dissagaliyeva/sim2bids"><img alt="Last commit" src="https://img.shields.io/github/last-commit/dissagaliyeva/sim2bids?style=plastic" /></a></li>
</ul>
<h3 id="events"><a class="toclink" href="#events">events</a></h3>
<ul>
<li><img src='./images/logo_python.png' width='14px'>  <a href="https://brainlife.io/docs/using_ezBIDS/">ezBIDS</a>: A web-based BIDS conversion tool with four unique features: (1) No installation or programming requirements. (2) Handling of both imaging and task events data and metadata. (3) Semi-automated inference and guidance for adherence to BIDS. (4) Multiple data management options, including download BIDS data to local system, or transfer to OpenNeuro.org or to brainlife.io.
  <br><img alt="" src="https://img.shields.io/badge/Javascript-yellow.svg?style=plastic" /><img alt="" src="https://img.shields.io/badge/shell-black.svg?style=plastic" /><a href="https://github.com/brainlife/ezbids"><img alt="Last commit" src="https://img.shields.io/github/last-commit/brainlife/ezbids?style=plastic" /></a></li>
</ul>
<h3 id="ieeg"><a class="toclink" href="#ieeg">iEEG</a></h3>
<ul>
<li><img src='./images/logo_matlab.png' width='17px'>  <a href="https://www.fieldtriptoolbox.org/example/bids/">FieldTrip - data2bids</a>:
  <br><a href="https://github.com/fieldtrip/fieldtrip"><img alt="Last commit" src="https://img.shields.io/github/last-commit/fieldtrip/fieldtrip?style=plastic" /></a></li>
<li><img src='./images/logo_python.png' width='14px'>  <a href="https://mne.tools/mne-bids">MNE-BIDS</a>: MNE-BIDS is a Python package that allows you to read and write BIDS-compatible datasets with the help of MNE-Python.
  <br><a href="https://github.com/mne-tools/mne-bids"><img alt="Last commit" src="https://img.shields.io/github/last-commit/mne-tools/mne-bids?style=plastic" /></a><a href="https://pypi.org/project/mne-bids"><img alt="PyPI version" src="https://badge.fury.io/py/mne-bids.svg" /></a></li>
</ul>
<h3 id="physiological"><a class="toclink" href="#physiological">physiological</a></h3>
<ul>
<li><img src='./images/logo_python.png' width='14px'>  <a href="https://bidscoin.readthedocs.io/en/stable/">BIDScoin</a>: BIDScoin is a flexible tool to convert (“coin”) source-level (raw) neuroimaging data sets to BIDS without needing to code anything. It features automatic data discovery based on header as well as filesystem information, and comes with a user-friendly GUI to add missing information and tweak the results. BIDScoin supports multiple source data formats with plugins (e.g. employing dcm2niix, spec2nii or nibabel) and allows customization of the prior knowledge about your data (allowing for fully automatic CLI data conversion if you like).
  <br><a href="https://github.com/Donders-Institute/bidscoin"><img alt="Last commit" src="https://img.shields.io/github/last-commit/Donders-Institute/bidscoin?style=plastic" /></a><a href="https://pypi.org/project/bidscoin"><img alt="PyPI version" src="https://badge.fury.io/py/bidscoin.svg" /></a></li>
<li><img src='./images/logo_python.png' width='14px'>  <a href="None">bidsphysio</a>: Converts physio data to BIDS physiological recording
  <br><a href="https://github.com/cbinyu/bidsphysio"><img alt="Last commit" src="https://img.shields.io/github/last-commit/cbinyu/bidsphysio?style=plastic" /></a><a href="https://pypi.org/project/bidsphysio/"><img alt="PyPI version" src="https://badge.fury.io/py/bidsphysio.svg" /></a><a href="https://hub.docker.com/r/cbinyu/bidsphysio/"><img alt="Docker version" src="https://img.shields.io/docker/pulls/cbinyu/bidsphysio.svg?style=plastic" /></a></li>
<li><img src='./images/logo_python.png' width='14px'>  <a href="https://phys2bids.readthedocs.io/en/latest/">phys2bids</a>: Python3 library to format physiological files in BIDS.
  <br><a href="https://github.com/physiopy/phys2bids"><img alt="Last commit" src="https://img.shields.io/github/last-commit/physiopy/phys2bids?style=plastic" /></a><a href="https://pypi.org/project/phys2bids"><img alt="PyPI version" src="https://badge.fury.io/py/phys2bids.svg" /></a></li>
</ul>
<!-- Converters ends -->

<h2 id="validation"><a class="toclink" href="#validation">Validation</a></h2>
<p>Make sure you use to validate any BIDS dataset you are working with.</p>
<p>You can <a href="https://bids-standard.github.io/bids-validator/">use it in a browser</a>
or <a href="https://github.com/bids-standard/bids-validator#quickstart">install the package</a>
and use it as a command line tool.</p>
<h2 id="bids-apps"><a class="toclink" href="#bids-apps">BIDS Apps</a></h2>
<p>BIDS apps are containerized tools to automatically process BIDS datasets.
For more information check the <a href="https://bids-apps.neuroimaging.io/apps/">BIDS Apps website</a>.</p>
<!-- APP starts -->
<ul>
<li><a href="https://github.com/bids-apps/afni_proc">afni_proc</a>: prototype AFNI bids app implementing participant level preprocessing with afni_proc.py
  <br><a href="https://hub.docker.com/r/bids/afni_proc"><img alt="Docker version" src="https://img.shields.io/docker/pulls/bids/afni_proc.svg?style=plastic" /></a></li>
<li><a href="https://github.com/bids-apps/antsCorticalThickness">antsCorticalThickness</a>: BIDS App for calculating cortical thickness using ANTs
  <br><a href="https://hub.docker.com/r/bids/antscorticalthickness"><img alt="Docker version" src="https://img.shields.io/docker/pulls/bids/antscorticalthickness.svg?style=plastic" /></a></li>
<li><a href="https://github.com/bids-apps/baracus">baracus</a>: Predicts brain age, based on data from Freesurfer 5.3
  <br><a href="https://hub.docker.com/r/bids/baracus"><img alt="Docker version" src="https://img.shields.io/docker/pulls/bids/baracus.svg?style=plastic" /></a></li>
<li><a href="https://github.com/bids-apps/brainiak-srm">brainiak-srm</a>: This is the BIDS-app version of the Shared Response Model (SRM) of BrainIAK
  <br><a href="https://hub.docker.com/r/bids/brainiak-srm"><img alt="Docker version" src="https://img.shields.io/docker/pulls/bids/brainiak-srm.svg?style=plastic" /></a></li>
<li><a href="https://github.com/bids-apps/BrainSuite">BrainSuite</a>: BrainSuite's structural, diffusion, and functional MRI processing pipelines with QC functionalities.
  <br><a href="https://hub.docker.com/r/bids/brainsuite"><img alt="Docker version" src="https://img.shields.io/docker/pulls/bids/brainsuite.svg?style=plastic" /></a></li>
<li><a href="https://github.com/bids-apps/BROCCOLI">BROCCOLI</a>: BIDS App for BROCCOLI
  <br><a href="https://hub.docker.com/r/bids/broccoli"><img alt="Docker version" src="https://img.shields.io/docker/pulls/bids/broccoli.svg?style=plastic" /></a></li>
<li><a href="https://github.com/bids-apps/CPAC">CPAC</a>: BIDS Application for the Configurable Pipeline for the Analysis of Connectomes (C-PAC)
  <br><a href="https://hub.docker.com/r/bids/cpac"><img alt="Docker version" src="https://img.shields.io/docker/pulls/bids/cpac.svg?style=plastic" /></a></li>
<li><a href="https://github.com/bids-apps/DPARSF">DPARSF</a>: Docker version of DPARSF, also deployed at OpenNeuro.org
  <br><a href="https://hub.docker.com/r/bids/dparsf"><img alt="Docker version" src="https://img.shields.io/docker/pulls/bids/dparsf.svg?style=plastic" /></a></li>
<li><a href="https://github.com/bids-apps/freesurfer">freesurfer</a>: BIDS app wrapping recon-all from FreeSurfer
  <br><a href="https://hub.docker.com/r/bids/freesurfer"><img alt="Docker version" src="https://img.shields.io/docker/pulls/bids/freesurfer.svg?style=plastic" /></a></li>
<li><a href="https://github.com/bids-apps/giga_connectome">giga_connectome</a>: This is a BIDS-App to extract signal from a parcellation with nilearn, typically useful in a context of resting-state data processing.
  <br><a href="https://hub.docker.com/r/bids/giga_connectome"><img alt="Docker version" src="https://img.shields.io/docker/pulls/bids/giga_connectome.svg?style=plastic" /></a></li>
<li><a href="https://github.com/bids-apps/HCPPipelines">HCPPipelines</a>: A BIDS App for minimal preprocessing using the HCP Pipelines
  <br><a href="https://hub.docker.com/r/bids/hcppipelines"><img alt="Docker version" src="https://img.shields.io/docker/pulls/bids/hcppipelines.svg?style=plastic" /></a></li>
<li><a href="https://github.com/bids-apps/hyperalignment">hyperalignment</a>: Hyperalignment is a functional alignment method that aligns subjects' brain data in a high-dimensional space of voxels/features.
  <br><a href="https://hub.docker.com/r/bids/hyperalignment"><img alt="Docker version" src="https://img.shields.io/docker/pulls/bids/hyperalignment.svg?style=plastic" /></a></li>
<li><a href="https://github.com/bids-apps/mindboggle">mindboggle</a>: Improve the accuracy, precision, and consistency of automated labeling and shape analysis of human brain image data
  <br><a href="https://hub.docker.com/r/bids/mindboggle"><img alt="Docker version" src="https://img.shields.io/docker/pulls/bids/mindboggle.svg?style=plastic" /></a></li>
<li><a href="https://github.com/bids-apps/MRtrix3_connectome">MRtrix3_connectome</a>: Generate subject connectomes from raw BIDS data &amp; perform inter-subject connection density normalization, using  the MRtrix3 software package.
  <br><a href="https://hub.docker.com/r/bids/mrtrix3_connectome"><img alt="Docker version" src="https://img.shields.io/docker/pulls/bids/mrtrix3_connectome.svg?style=plastic" /></a></li>
<li><a href="https://github.com/bids-apps/ndmg">ndmg</a>: BIDS app for NeuroData's MRI to Graphs pipeline
  <br><a href="https://hub.docker.com/r/bids/ndmg"><img alt="Docker version" src="https://img.shields.io/docker/pulls/bids/ndmg.svg?style=plastic" /></a></li>
<li><a href="https://github.com/bids-apps/nipypelines">nipypelines</a>: Preprocess functional tasks in a BIDS dataset.
  <br><a href="https://hub.docker.com/r/bids/nipypelines"><img alt="Docker version" src="https://img.shields.io/docker/pulls/bids/nipypelines.svg?style=plastic" /></a></li>
<li><a href="https://github.com/bids-apps/oppni">oppni</a>: runs fast optimization of preprocessing pipelines for BOLD fMRI
  <br><a href="https://hub.docker.com/r/bids/oppni"><img alt="Docker version" src="https://img.shields.io/docker/pulls/bids/oppni.svg?style=plastic" /></a></li>
<li><a href="https://github.com/bids-apps/PyMVPA">PyMVPA</a>: runs MVPA and RSA analysis BIDS bold derivative data
  <br><a href="https://hub.docker.com/r/bids/pymvpa"><img alt="Docker version" src="https://img.shields.io/docker/pulls/bids/pymvpa.svg?style=plastic" /></a></li>
<li><a href="https://github.com/bids-apps/QAP">QAP</a>: PCP Quality Assessment Protocol
  <br><a href="https://hub.docker.com/r/bids/qap"><img alt="Docker version" src="https://img.shields.io/docker/pulls/bids/qap.svg?style=plastic" /></a></li>
<li><a href="https://github.com/bids-apps/rsHRF">rsHRF</a>: Resting state HRF estimation from BOLD-fMRI signal
  <br><a href="https://hub.docker.com/r/bids/rshrf"><img alt="Docker version" src="https://img.shields.io/docker/pulls/bids/rshrf.svg?style=plastic" /></a></li>
<li><a href="https://github.com/bids-apps/SPM">SPM</a>: BIDS App containing an instance of the SPM software.
  <br><a href="https://hub.docker.com/r/bids/spm"><img alt="Docker version" src="https://img.shields.io/docker/pulls/bids/spm.svg?style=plastic" /></a></li>
<li><a href="https://github.com/bids-apps/tracula">tracula</a>: implements Freesurfer's TRACULA (TRActs Constrained by UnderLying Anatomy) tool for cross-sectional as well as longitudinal (multi session) input data.
  <br><a href="https://hub.docker.com/r/bids/tracula"><img alt="Docker version" src="https://img.shields.io/docker/pulls/bids/tracula.svg?style=plastic" /></a></li>
<li><a href="https://github.com/connectomicslab/connectomemapper3">connectomemapper3</a>: Connectome Mapper 3 is a BIDS App that implements full anatomical, diffusion, resting/state functional MRI, and recently EEG processing pipelines, from raw T1 / DWI / BOLD , and preprocessed EEG data to multi-resolution brain parcellation with corresponding connection matrices.
  <br><a href="https://hub.docker.com/r/sebastientourbier/connectomemapper-bidsapp"><img alt="Docker version" src="https://img.shields.io/docker/pulls/sebastientourbier/connectomemapper-bidsapp.svg?style=plastic" /></a></li>
<li><a href="https://github.com/cpp-lln-lab/bidsMReye">bidsMReye</a>: BIDS app using deepMReye to decode eye motion for fMRI time series data.
  <br><a href="https://hub.docker.com/r/cpplab/bidsmreye"><img alt="Docker version" src="https://img.shields.io/docker/pulls/cpplab/bidsmreye.svg?style=plastic" /></a></li>
<li><a href="https://github.com/cpp-lln-lab/bidspm">bidspm</a>: an SPM centric BIDS app
  <br><a href="https://hub.docker.com/r/cpplab/bidspm"><img alt="Docker version" src="https://img.shields.io/docker/pulls/cpplab/bidspm.svg?style=plastic" /></a></li>
<li><a href="https://github.com/DCAN-Labs/BIBSnet">BIBSnet</a>: Utility for creating a nnU-Net anatomical MRI segmentation and mask with a infant brain trained model for the purposes of circumventing JLF within Nibabies.
  <br><a href="https://hub.docker.com/r/dcanumn/bibsnet"><img alt="Docker version" src="https://img.shields.io/docker/pulls/dcanumn/bibsnet.svg?style=plastic" /></a></li>
<li><a href="https://github.com/djarecka/fmriprep-fake">fmriprep-fake</a>: None
  <br><a href="https://hub.docker.com/r/djarecka/fmriprep_fake"><img alt="Docker version" src="https://img.shields.io/docker/pulls/djarecka/fmriprep_fake.svg?style=plastic" /></a></li>
<li><a href="https://github.com/khanlab/funcmasker-flex">funcmasker-flex</a>: BIDS App for U-net brain masking of fetal bold MRI
  <br><a href="https://hub.docker.com/r/khanlab/funcmasker-flex"><img alt="Docker version" src="https://img.shields.io/docker/pulls/khanlab/funcmasker-flex.svg?style=plastic" /></a></li>
<li><a href="https://github.com/khanlab/hippunfold">hippunfold</a>: BIDS App for Hippunfold (automated hippocampal unfolding and subfield segmentation)
  <br><a href="https://hub.docker.com/r/khanlab/hippunfold"><img alt="Docker version" src="https://img.shields.io/docker/pulls/khanlab/hippunfold.svg?style=plastic" /></a></li>
<li><a href="https://github.com/Medical-Image-Analysis-Laboratory/mialsuperresolutiontoolkit">mialsuperresolutiontoolkit</a>: The Medical Image Analysis Laboratory Super-Resolution ToolKit (MIALSRTK) consists of a set of C++ and Python processing and workflow tools necessary to perform motion-robust super-resolution fetal MRI reconstruction in the BIDS Apps framework.
  <br><a href="https://hub.docker.com/r/sebastientourbier/mialsuperresolutiontoolkit"><img alt="Docker version" src="https://img.shields.io/docker/pulls/sebastientourbier/mialsuperresolutiontoolkit.svg?style=plastic" /></a></li>
<li><a href="https://github.com/MICA-MNI/micapipe">micapipe</a>: micapipe from the Multimodal imaging and connectome analysis lab (https://mica-mni.github.io) at the Montreal Neurological Institute. Read The Docs documentation below
  <br><a href="https://hub.docker.com/r/micalab/micapipe"><img alt="Docker version" src="https://img.shields.io/docker/pulls/micalab/micapipe.svg?style=plastic" /></a></li>
<li><a href="https://github.com/nipreps/dmriprep">dmriprep</a>: dMRIPrep is a robust and easy-to-use pipeline for preprocessing of diverse dMRI data. The transparent workflow dispenses of manual intervention, thereby ensuring the reproducibility of the results.
  <br><a href="https://hub.docker.com/r/nipreps/dmriprep"><img alt="Docker version" src="https://img.shields.io/docker/pulls/nipreps/dmriprep.svg?style=plastic" /></a></li>
<li><a href="https://github.com/nipreps/fmripost-aroma">fmripost-aroma</a>: Functional MRI postprocessing with ICA-AROMA
  <br><a href="https://hub.docker.com/r/nipreps/fmripost-aroma"><img alt="Docker version" src="https://img.shields.io/docker/pulls/nipreps/fmripost-aroma.svg?style=plastic" /></a></li>
<li><a href="https://github.com/nipreps/fmripost-phase">fmripost-phase</a>: Postprocessing of complex-valued fMRI data
  <br><a href="https://hub.docker.com/r/nipreps/fmripost-phase"><img alt="Docker version" src="https://img.shields.io/docker/pulls/nipreps/fmripost-phase.svg?style=plastic" /></a></li>
<li><a href="https://github.com/nipreps/fmripost-rapidtide">fmripost-rapidtide</a>: Functional MRI postprocessing with Rapidtide
  <br><a href="https://hub.docker.com/r/nipreps/fmripost-rapidtide"><img alt="Docker version" src="https://img.shields.io/docker/pulls/nipreps/fmripost-rapidtide.svg?style=plastic" /></a></li>
<li><a href="https://github.com/nipreps/fmriprep">fmriprep</a>: fMRIPrep is a robust and easy-to-use pipeline for preprocessing of diverse fMRI data. The transparent workflow dispenses of manual intervention, thereby ensuring the reproducibility of the results.
  <br><a href="https://hub.docker.com/r/nipreps/fmriprep"><img alt="Docker version" src="https://img.shields.io/docker/pulls/nipreps/fmriprep.svg?style=plastic" /></a></li>
<li><a href="https://github.com/nipreps/mriqc">mriqc</a>: Automated Quality Control and visual reports for Quality Assessment of structural (T1w, T2w) and functional MRI of the brain
  <br><a href="https://hub.docker.com/r/nipreps/mriqc"><img alt="Docker version" src="https://img.shields.io/docker/pulls/nipreps/mriqc.svg?style=plastic" /></a></li>
<li><a href="https://github.com/nipreps/nibabies">nibabies</a>: Processing tools for magnetic resonance images of infant brains
  <br><a href="https://hub.docker.com/r/nipreps/nibabies"><img alt="Docker version" src="https://img.shields.io/docker/pulls/nipreps/nibabies.svg?style=plastic" /></a></li>
<li><a href="https://github.com/nipreps/nirodents">nirodents</a>: None
  <br><a href="https://hub.docker.com/r/nipreps/nirodents"><img alt="Docker version" src="https://img.shields.io/docker/pulls/nipreps/nirodents.svg?style=plastic" /></a></li>
<li><a href="https://github.com/nipreps/smriprep">smriprep</a>: Structural MRI PREProcessing (sMRIPrep) workflows for NIPreps (NeuroImaging PREProcessing tools)
  <br><a href="https://hub.docker.com/r/nipreps/smriprep"><img alt="Docker version" src="https://img.shields.io/docker/pulls/nipreps/smriprep.svg?style=plastic" /></a></li>
<li><a href="https://github.com/openneuropet/petdeface">petdeface</a>: A nipype implementation of MiDeface used to deface PET and MR data as well as co-register the two modalities.
  <br><a href="https://hub.docker.com/r/openneuropet/petdeface"><img alt="Docker version" src="https://img.shields.io/docker/pulls/openneuropet/petdeface.svg?style=plastic" /></a></li>
<li><a href="https://github.com/PeerHerholz/BIDSonym">BIDSonym</a>: a BIDS app for pseudo-anonymization of neuroimaging data
  <br><a href="https://hub.docker.com/r/peerherholz/bidsonym"><img alt="Docker version" src="https://img.shields.io/docker/pulls/peerherholz/bidsonym.svg?style=plastic" /></a></li>
<li><a href="https://github.com/PennLINC/aslprep">aslprep</a>: Preprocessing of arterial spin labeling data
  <br><a href="https://hub.docker.com/r/pennlinc/aslprep"><img alt="Docker version" src="https://img.shields.io/docker/pulls/pennlinc/aslprep.svg?style=plastic" /></a></li>
<li><a href="https://github.com/PennLINC/qsiprep">qsiprep</a>: Preprocessing of diffusion MRI
  <br><a href="https://hub.docker.com/r/pennbbl/qsiprep"><img alt="Docker version" src="https://img.shields.io/docker/pulls/pennbbl/qsiprep.svg?style=plastic" /></a></li>
<li><a href="https://github.com/PennLINC/qsirecon">qsirecon</a>: Reconstruction of preprocessed diffusion MRI
  <br><a href="https://hub.docker.com/r/pennlinc/qsirecon"><img alt="Docker version" src="https://img.shields.io/docker/pulls/pennlinc/qsirecon.svg?style=plastic" /></a></li>
<li><a href="https://github.com/PennLINC/xcp_d">xcp_d</a>: Post-processing of fMRIPrep, nibabies, HCP, and DCAN BOLD derivatives for functional connectivity analysis
  <br><a href="https://hub.docker.com/r/pennlinc/xcp_d"><img alt="Docker version" src="https://img.shields.io/docker/pulls/pennlinc/xcp_d.svg?style=plastic" /></a></li>
<li><a href="https://github.com/poldracklab/fitlins">fitlins</a>: Fit Linear Models to BIDS Datasets
  <br><a href="https://hub.docker.com/r/poldracklab/fitlins"><img alt="Docker version" src="https://img.shields.io/docker/pulls/poldracklab/fitlins.svg?style=plastic" /></a></li>
<li><a href="https://github.com/reprostat/reproa">reproa</a>: BIDS App containing an instance of the ReproAnalysis (reproa) software (core only) running under Octave with minimum dependencies.
  <br><a href="https://hub.docker.com/r/reprostat/reproa"><img alt="Docker version" src="https://img.shields.io/docker/pulls/reprostat/reproa.svg?style=plastic" /></a></li>
<li><a href="https://github.com/sebastientourbier/multiscalebrainparcellator">multiscalebrainparcellator</a>: Multi-Scale Brain Parcellator, part of the Connectome Mapping Toolkit (CMTK), is a BIDS App that implements a full anatomical MRI processing pipeline, from raw T1w data to structural brain parcellation at five different scales.
  <br><a href="https://hub.docker.com/r/sebastientourbier/multiscalebrainparcellator"><img alt="Docker version" src="https://img.shields.io/docker/pulls/sebastientourbier/multiscalebrainparcellator.svg?style=plastic" /></a></li>
<li><a href="https://github.com/trendscenter/gift-bids">gift-bids</a>: None
  <br><a href="https://hub.docker.com/r/trends/gift-bids"><img alt="Docker version" src="https://img.shields.io/docker/pulls/trends/gift-bids.svg?style=plastic" /></a></li>
</ul>
<!-- APP ends -->

<h2 id="tools"><a class="toclink" href="#tools">Tools</a></h2>
<p>Software packages for working with BIDS datasets.</p>
<!-- TOOLS starts -->
<ul>
<li><img src='./images/logo_python.png' width='14px'> <a href="https://ancpbids.readthedocs.io/en/latest/">ancp-bids</a>: A Python package to read/write/query/validate BIDS datasets.
  <br><a href="https://github.com/ANCPLabOldenburg/ancp-bids"><img alt="Last commit" src="https://img.shields.io/github/last-commit/ANCPLabOldenburg/ancp-bids?style=plastic" /></a> <a href="https://pypi.org/project/ancpbids/"><img alt="PyPI version" src="https://badge.fury.io/py/ancpbids.svg" /></a></li>
<li><img src='./images/logo_python.png' width='14px'> <a href="https://pennlinc-babs.readthedocs.io/">babs</a>: BIDS App Bootstrap (BABS) is a reproducible, generalizable, and scalable Python package for BIDS App analysis of large datasets. It uses DataLad and adopts FAIRly big framework.
  <br><a href="https://github.com/PennLINC/babs"><img alt="Last commit" src="https://img.shields.io/github/last-commit/PennLINC/babs?style=plastic" /></a> <a href="https://pypi.org/project/babs/"><img alt="PyPI version" src="https://badge.fury.io/py/babs.svg" /></a></li>
<li><img src='./images/logo_python.png' width='14px'> <a href="https://bids-standard.github.io/stats-models/index.html">bids stats model</a>: Validate BIDS statistical model. To learn more the <a href="https://bids-standard.github.io/stats-models/index.html">BIDS stats model website</a>
  <br><a href="https://github.com/bids-standard/stats-models"><img alt="Last commit" src="https://img.shields.io/github/last-commit/bids-standard/stats-models?style=plastic" /></a> <a href="https://pypi.org/project/bsmschema/"><img alt="PyPI version" src="https://badge.fury.io/py/bsmschema.svg" /></a></li>
<li><img src='./images/logo_python.png' width='14px'> <a href="https://gitlab.indiscale.com/caosdb/src/crawler-cfoods/bids-cfood">bids-cfood</a>: a module to handle BIDS dataset for the caosDB data crawler</li>
<li><img src='./images/logo_matlab.png' width='17px'> <img src='./images/logo_octave.png' width='16px'> <a href="https://bids-matlab.readthedocs.io/en/main/">bids-matlab</a>: MATLAB/Octave tools to interact with datasets conforming to the BIDS format
  <br><a href="https://github.com/bids-standard/bids-matlab"><img alt="Last commit" src="https://img.shields.io/github/last-commit/bids-standard/bids-matlab?style=plastic" /></a></li>
<li><img src='./images/logo_python.png' width='14px'> <a href="https://github.com/gold-standard-phantoms/bids-pydantic">BIDS-pydantic</a>: Pulls a specified version of the BIDS schema and creates corresponding Pydantic models, which will provide BIDS data validation using Python type annotations. See also <a href="https://pypi.org/project/BIDS-pydantic-models/">BIDS-pydantic-models</a>.
  <br><a href="https://github.com/gold-standard-phantoms/bids-pydantic"><img alt="Last commit" src="https://img.shields.io/github/last-commit/gold-standard-phantoms/bids-pydantic?style=plastic" /></a> <a href="https://pypi.org/project/BIDS-pydantic/"><img alt="PyPI version" src="https://badge.fury.io/py/BIDS-pydantic.svg" /></a></li>
<li><img src='./images/logo_python.png' width='14px'> <a href="https://bids2cite.readthedocs.io/en/latest/">bids2cite</a>: package to interactively update <code>dataset_description.json</code> and generate citation files (for example <code>datacite.yml</code>) for BIDS datasets.
  <br><a href="https://github.com/Remi-Gau/bids2cite"><img alt="Last commit" src="https://img.shields.io/github/last-commit/Remi-Gau/bids2cite?style=plastic" /></a></li>
<li><img src='./images/logo_python.png' width='14px'> <a href="https://childmindresearch.github.io/bids2table/bids2table.html">bids2table</a>: bids2table is a library for efficiently indexing and querying large-scale BIDS neuroimaging datasets and derivatives.
  <br><a href="https://github.com/childmindresearch/bids2table"><img alt="Last commit" src="https://img.shields.io/github/last-commit/childmindresearch/bids2table?style=plastic" /></a> <a href="https://pypi.org/project/bids2table/"><img alt="PyPI version" src="https://badge.fury.io/py/bids2table.svg" /></a></li>
<li><img src='./images/logo_R.png' width='18px'> <a href="https://bbuchsbaum.github.io/bidser/">bidser</a>: Working with Brain Imaging Data Structure in R
  <br><a href="https://github.com/bbuchsbaum/bidser"><img alt="Last commit" src="https://img.shields.io/github/last-commit/bbuchsbaum/bidser?style=plastic" /></a></li>
<li><img src='./images/logo_python.png' width='14px'> <a href="https://github.com/Macquarie-MEG-Research/BIDSHandler">BIDSHandler</a>: Library for loading and manipulating BIDS compatible MEG data
  <br><a href="https://github.com/Macquarie-MEG-Research/BIDSHandler"><img alt="Last commit" src="https://img.shields.io/github/last-commit/Macquarie-MEG-Research/BIDSHandler?style=plastic" /></a></li>
<li><img src='./images/logo_matlab.png' width='17px'> <a href="http://neuroimage.usc.edu/brainstorm/">Brainstorm</a>: MEG/EEG analysis package
  <br><a href="https://github.com/brainstorm-tools/brainstorm3"><img alt="Last commit" src="https://img.shields.io/github/last-commit/brainstorm-tools/brainstorm3?style=plastic" /></a></li>
<li><img src='./images/logo_python.png' width='14px'> <a href="https://clpipe.readthedocs.io/en/latest/index.html">clpipe</a>: streamlined processing pipeline for MRI data centered around BIDS
  <br><a href="https://github.com/cohenlabUNC/clpipe"><img alt="Last commit" src="https://img.shields.io/github/last-commit/cohenlabUNC/clpipe?style=plastic" /></a></li>
<li><img src='./images/logo_python.png' width='14px'> <a href="https://cubids.readthedocs.io/en/latest/">cuBIDS</a>: a Python package designed to facilitate reproducible curation of neuroimaging BIDS datasets
  <br><a href="https://github.com/pennlinc/cubids"><img alt="Last commit" src="https://img.shields.io/github/last-commit/pennlinc/cubids?style=plastic" /></a> <a href="https://pypi.org/project/cubids/"><img alt="PyPI version" src="https://badge.fury.io/py/cubids.svg" /></a></li>
<li><img src='./images/logo_python.png' width='14px'> <a href="https://github.com/DCAN-Labs/file-mapper">File mapper</a>: An easy tool to copy/move/symlink files from one directory to the other! Can be used to "convert" dataset to be BIDS compliant.
  <br><a href="https://github.com/DCAN-Labs/file-mapper"><img alt="Last commit" src="https://img.shields.io/github/last-commit/DCAN-Labs/file-mapper?style=plastic" /></a></li>
<li><img src='./images/logo_python.png' width='14px'> <a href="https://github.com/tolik-g/BIDS">GUI dataset description generator</a>: GUI form that generates <code>dataset_description.json</code>
  <br><a href="https://github.com/tolik-g/BIDS"><img alt="Last commit" src="https://img.shields.io/github/last-commit/tolik-g/BIDS?style=plastic" /></a></li>
<li><img src='./images/logo_python.png' width='14px'> <a href="https://github.com/HALFpipe/HALFpipe?tab=readme-ov-file#getting-started">HALFpipe</a>: wrapper for fmriprep and commong resting-stat and statistical analysis for fMRI
  <br><a href="https://github.com/HALFpipe/HALFpipe"><img alt="Last commit" src="https://img.shields.io/github/last-commit/HALFpipe/HALFpipe?style=plastic" /></a></li>
<li><a href="https://www.hed-resources.org">Hierarchical Event Descriptors (HED) online tools</a>: Online tools for annotation, validation, summary, and assembly of event file contents and annotations.</li>
<li><img src='./images/logo_python.png' width='14px'> <a href="https://www.hed-resources.org">Hierarchical Event Descriptors (HED) Python tools</a>: HED libraries supporting schema development as well as annotation, validation, and analysis.
  <br><a href="https://github.com/hed-standard/hed-Python"><img alt="Last commit" src="https://img.shields.io/github/last-commit/hed-standard/hed-Python?style=plastic" /></a> <a href="https://pypi.org/project/hedtools/"><img alt="PyPI version" src="https://badge.fury.io/py/hedtools.svg" /></a></li>
<li><img src='./images/logo_python.png' width='14px'> <a href="https://lazybids.github.io/lazybids/">LazyBIDS</a>: Python package to interact with BIDS datasets in a pythonic way. A BIDS dataset is read to hierarchical pydantic models (e.g. dataset-&gt;subjects-&gt;sessions-&gt;scans), variables are accesibles as model properties, tables, SimpleITK Images and numpy arrays. Also serves as python client for lazybids-ui server.
  <br><a href="https://github.com/lazybids/lazybids"><img alt="Last commit" src="https://img.shields.io/github/last-commit/lazybids/lazybids?style=plastic" /></a> <a href="https://pypi.org/project/lazybids/"><img alt="PyPI version" src="https://badge.fury.io/py/lazybids.svg" /></a></li>
<li><img src='./images/logo_python.png' width='14px'> <a href="https://lazybids.github.io/lazybids-ui/">LazyBIDS-ui</a>: FastAPI web server providing a GUI to explore (imaging) BIDS datasets interactively, also provides (XNAT-like) REST-API to access (parts of) datasets on a client machine. Uses NiiVue to view and navigate (3D) images in the browser.
  <br><a href="https://github.com/lazybids/lazybids-ui"><img alt="Last commit" src="https://img.shields.io/github/last-commit/lazybids/lazybids-ui?style=plastic" /></a> <a href="https://pypi.org/project/lazybids-ui/"><img alt="PyPI version" src="https://badge.fury.io/py/lazybids-ui.svg" /></a></li>
<li><img src='./images/logo_matlab.png' width='17px'> <a href="https://www.lead-dbs.org/">Lead-DBS</a>: A toolbox facilitating Deep Brain Stimulation electrode reconstructions and computer simulations supports BIDS conversion and ingestion of BIDS datasets.
  <br><a href="https://github.com/netstim/leaddbs"><img alt="Last commit" src="https://img.shields.io/github/last-commit/netstim/leaddbs?style=plastic" /></a></li>
<li><img src='./images/logo_python.png' width='14px'> <a href="https://mne.tools/mne-bids/stable/index.html">mne-bids</a>: collection of tools for converting magnetoencephalography (MEG) data into BIDS format, as well as some helper functions for creating the folders and metadata needed for a BIDS dataset.
  <br><a href="https://github.com/mne-tools/mne-bids"><img alt="Last commit" src="https://img.shields.io/github/last-commit/mne-tools/mne-bids?style=plastic" /></a></li>
<li><img src='./images/logo_python.png' width='14px'> <a href="https://mne.tools/mne-bids-pipeline/stable/index.html">mne-bids-pipeline</a>: MNE-BIDS-Pipeline is a full-flegded processing pipeline for your MEG and EEG data. Under the hood, it uses MNE-Python.
  <br><a href="https://github.com/mne-tools/mne-bids-pipeline"><img alt="Last commit" src="https://img.shields.io/github/last-commit/mne-tools/mne-bids-pipeline?style=plastic" /></a></li>
<li><a href="https://neurobagel.org/data_prep/">neurobagel annotate</a>: This tool allows you to create a machine readable data dictionary in .json format for a tabular phenotypic file in .tsv format ("Data table").
  <br><a href="https://github.com/neurobagel/annotation_tool"><img alt="Last commit" src="https://img.shields.io/github/last-commit/neurobagel/annotation_tool?style=plastic" /></a></li>
<li><a href="https://neurobagel.org/public_nodes/">neurobagel query</a>: Neurobagel's query tool is a web interface for searching across a BIDS datasets based on various subject clinical-demographic and imaging parameters.
  <br><a href="https://github.com/neurobagel/query-tool"><img alt="Last commit" src="https://img.shields.io/github/last-commit/neurobagel/query-tool?style=plastic" /></a></li>
<li><img src='./images/logo_python.png' width='14px'> <a href="https://nipoppy.readthedocs.io/en/latest/">nipopy</a>: Lightweight neuroimaging workflow manager to help with DICOM to BIDS conversion and running BIDS apps.
  <br><a href="https://github.com/nipoppy/nipoppy"><img alt="Last commit" src="https://img.shields.io/github/last-commit/nipoppy/nipoppy?style=plastic" /></a></li>
<li><a href="https://github.com/vistalab/PRFmodel/wiki">PRFmodel</a>: a set of tools to fit population receptive field models to BIDS datasets
  <br><a href="https://github.com/vistalab/PRFmodel"><img alt="Last commit" src="https://img.shields.io/github/last-commit/vistalab/PRFmodel?style=plastic" /></a></li>
<li><img src='./images/logo_python.png' width='14px'> <a href="https://psychopy-bids.readthedocs.io/en/latest/">psychopy-bids</a>: A psychopy plugin to help easily output a BIDS dataset, including <code>events.tsv</code> and <code>beh.tsv</code> files when running experiments with psychopy.</li>
<li><img src='./images/logo_python.png' width='14px'> <a href="https://bids-standard.github.io/pybids/">PyBIDS</a>: Python package to quickly parse / search the components of a BIDS dataset. It also contains functionality for running analyses on your data.
  <br><a href="https://github.com/bids-standard/pybids"><img alt="Last commit" src="https://img.shields.io/github/last-commit/bids-standard/pybids?style=plastic" /></a> <a href="https://pypi.org/project/pybids/"><img alt="PyPI version" src="https://badge.fury.io/py/pybids.svg" /></a></li>
<li><img src='./images/logo_R.png' width='18px'> <a href="https://github.com/mathesong/rbids">rbids</a>: aims to make BIDS datasets more easily accessible for packages written in R
  <br><a href="https://github.com/mathesong/rbids"><img alt="Last commit" src="https://img.shields.io/github/last-commit/mathesong/rbids?style=plastic" /></a></li>
<li><img src='./images/logo_matlab.png' width='17px'> <img src='./images/logo_octave.png' width='16px'> <a href="https://spm-2-bids.readthedocs.io/en/latest/">spm_2_bids</a>: a tool convert SPM preprocessed output to BIDS derivatives (trying to follow <a href="https://bids.neuroimaging.io/bep012">BEP12</a>)
  <br><a href="https://github.com/cpp-lln-lab/spm_2_bids"><img alt="Last commit" src="https://img.shields.io/github/last-commit/cpp-lln-lab/spm_2_bids?style=plastic" /></a></li>
</ul>
<!-- TOOLS ends -->

<h2 id="code"><a class="toclink" href="#code">Code</a></h2>
<p>Most of the repositories for BIDS are centralized in:</p>
<ul>
<li>the <a href="https://github.com/bids-standard">BIDS standard github organization</a></li>
<li>the <a href="https://github.com/bids-apps">BIDS-app github organization</a></li>
</ul>
<p>Make sure to also check repositories tagged for the following topics:</p>
<ul>
<li>
<p>on GitHub:</p>
</li>
<li>
<p><a href="https://github.com/topics/bids">BIDS</a></p>
</li>
<li><a href="https://github.com/topics/bidsapp">BIDS-Apps</a></li>
<li>
<p><a href="https://github.com/topics/bids-format">bids-format</a></p>
</li>
<li>
<p>on GitLab:</p>
</li>
<li>
<p><a href="https://gitlab.com/search?search=bids&amp;nav_source=navbar">BIDS</a></p>
</li>
</ul>
<h2 id="bids-extension-proposals-bep"><a class="toclink" href="#bids-extension-proposals-bep">BIDS Extension Proposals (BEP)</a></h2>
<p>Ongoing community proposals for extending the BIDS specification to new datatypes (<a href="#raw">raw</a>),
to specify how to organized processed data (<a href="#derivatives">derivatives</a>),
or interoperating with BIDS datasets.</p>
<p>Make sure to check the list to see if some people are not already working
on making sure BIDS supports your favorite datatype.</p>
<p>If you are working on an extension proposal
make sure to check our <a href="https://bids-extensions.readthedocs.io/">documentation regarding the BIDS extension proposal process</a>.</p>
<!-- BEP starts -->

<h3 id="raw"><a class="toclink" href="#raw">raw</a></h3>
<ul>
<li><a href="https://bids.neuroimaging.io/bep004">BEP004</a>: Susceptibility Weighted Imaging</li>
<li><a href="https://bids.neuroimaging.io/bep020">BEP020</a>: Eye Tracking including Gaze Position and Pupil Size</li>
<li><a href="https://bids.neuroimaging.io/bep024">BEP024</a>: Computed Tomography scan</li>
<li><a href="https://bids.neuroimaging.io/bep026">BEP026</a>: Microelectrode Recordings</li>
<li><a href="https://bids.neuroimaging.io/bep032">BEP032</a>: Animal electrophysiology</li>
<li><a href="https://bids.neuroimaging.io/bep033">BEP033</a>: Advanced Diffusion Weighted Imaging</li>
<li><a href="https://bids.neuroimaging.io/bep036">BEP036</a>: Phenotypic Data Guidelines</li>
<li><a href="https://bids.neuroimaging.io/bep037">BEP037</a>: Non-Invasive Brain Stimulation</li>
<li><a href="https://bids.neuroimaging.io/bep038">BEP038</a>: Atlases</li>
<li><a href="https://bids.neuroimaging.io/bep039">BEP039</a>: Dimensionality reduction-based networks</li>
<li><a href="https://bids.neuroimaging.io/bep040">BEP040</a>: Functional Ultrasound</li>
<li><a href="https://bids.neuroimaging.io/bep042">BEP042</a>: Electromyography</li>
<li><a href="https://bids.neuroimaging.io/bep044">BEP044</a>: Stimuli</li>
</ul>
<h3 id="derivative"><a class="toclink" href="#derivative">derivative</a></h3>
<ul>
<li><a href="https://bids.neuroimaging.io/bep011">BEP011</a>: Structural preprocessing derivatives</li>
<li><a href="https://bids.neuroimaging.io/bep012">BEP012</a>: Functional preprocessing derivatives</li>
<li><a href="https://bids.neuroimaging.io/bep014">BEP014</a>: Affine transformations and nonlinear field warps</li>
<li><a href="https://bids.neuroimaging.io/bep016">BEP016</a>: Diffusion weighted imaging derivatives</li>
<li><a href="https://bids.neuroimaging.io/bep017">BEP017</a>: Generic BIDS connectivity data schema</li>
<li><a href="https://bids.neuroimaging.io/bep021">BEP021</a>: Common Electrophysiological Derivatives</li>
<li><a href="https://bids.neuroimaging.io/bep023">BEP023</a>: PET Preprocessing derivatives</li>
<li><a href="https://bids.neuroimaging.io/bep034">BEP034</a>: Computational modeling</li>
<li><a href="https://bids.neuroimaging.io/bep035">BEP035</a>: Modular extensions for individual participant data mega-analyses with non-compliant derivatives</li>
<li><a href="https://bids.neuroimaging.io/bep041">BEP041</a>: Statistical Model Derivatives</li>
</ul>
<!-- BEP ends -->

<h2 id="publications"><a class="toclink" href="#publications">Publications</a></h2>
<p>BIDS references are centralized in <a href="https://www.zotero.org/groups/5111637/bids">our zotero group</a>.</p>
<h2 id="contribute"><a class="toclink" href="#contribute">Contribute</a></h2>
<p>Contributions welcome! Read the <a href="contributing.html">contribution guidelines</a> first.</p>












                
              </article>
            </div>
          
          
<script>var target=document.getElementById(location.hash.slice(1));target&&target.name&&(target.checked=target.name.startsWith("__tabbed_"))</script>
        </div>
        
          <button type="button" class="md-top md-icon" data-md-component="top" hidden>
  
  <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M13 20h-2V8l-5.5 5.5-1.42-1.42L12 4.16l7.92 7.92-1.42 1.42L13 8v12Z"/></svg>
  Back to top
</button>
        
      </main>
      
        <footer class="md-footer">
  
  <div class="md-footer-meta md-typeset">
    <div class="md-footer-meta__inner md-grid">
      <div class="md-copyright">
  
  
    Made with
    <a href="https://squidfunk.github.io/mkdocs-material/" target="_blank" rel="noopener">
      Material for MkDocs
    </a>
  
</div>
      
    </div>
  </div>
</footer>
      
    </div>
    <div class="md-dialog" data-md-component="dialog">
      <div class="md-dialog__inner md-typeset"></div>
    </div>
    
    
    <script id="__config" type="application/json">{"base": ".", "features": ["navigation.top", "content.action.edit"], "search": "assets/javascripts/workers/search.b8dbb3d2.min.js", "translations": {"clipboard.copied": "Copied to clipboard", "clipboard.copy": "Copy to clipboard", "search.result.more.one": "1 more on this page", "search.result.more.other": "# more on this page", "search.result.none": "No matching documents", "search.result.one": "1 matching document", "search.result.other": "# matching documents", "search.result.placeholder": "Type to start searching", "search.result.term.missing": "Missing", "select.version": "Select version"}}</script>
    
    
      <script src="assets/javascripts/bundle.1e8ae164.min.js"></script>
      
    
  </body>
</html>